target/linux/generic/config-6.12

```diff
@@ -7509,11 +7649,13 @@ CONFIG_XFRM=y
 # CONFIG_XMON is not set
 CONFIG_XZ_DEC=y
 # CONFIG_XZ_DEC_ARM is not set
+CONFIG_XZ_DEC_ARM64=y
```

Comment:
`CONFIG_XZ_DEC_ARM64` and `CONFIG_XZ_DEC_RISCV` should be disabled by default.

Reply:
Ok, I'll do it in the next push.